github_projects_board.md

```diff
@@ -0,0 +1,26 @@
+# GitHub Projects board
```

Comment:
@liatsade 

> what other columns can be added? for example, I have an "upcoming" column for my project.

"Waiting for client approval", "Approved by client"

> what columns are mandatory

Waiting for QA

> is it true that the AM assigns issues to the developers? Shouldn't the team lead do it?

Depends on of the issue, depends on of the AM and the Issues flow. 

> Who does what at each column is not clear - for example, who moves the issue to QA?

I instructed the person in charge of the task to move it by themselves, like a volleyball. The AM, is taking the role of the client in the columns flow.

> Are issues deleted from Merged column?

Merged used us as a label - not a column - it should be well defined when to use a label (I think of it as a stamp) and when to use the column (I think of it as a step/phase).